ArchiSteamFarm/Steam/Data/AccessTokenResponse.cs

```diff
@@ -0,0 +1,23 @@
+
+using System.Diagnostics.CodeAnalysis;
+using Newtonsoft.Json;
+using SteamKit2;
+
+namespace ArchiSteamFarm.Steam.Data {
+	[SuppressMessage("ReSharper", "ClassCannotBeInstantiated")]
+	internal sealed class AccessTokenResponse : EResultResponse {
+		[JsonProperty(PropertyName = "success", Required = Required.Always)]
+		internal EResult Success { get; private set; }
+
+		[JsonProperty(PropertyName = "data", Required = Required.Always)]
+		internal AccessTokenData? AccessTokenData { get; private set; }
```

Comment:
You have this as `Required = Required.Always`, so remove the null operator and initialize it with `new()`. Afterwards remove conditional access below.